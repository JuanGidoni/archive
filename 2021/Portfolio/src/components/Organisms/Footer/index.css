/*
****************
 *****************
   FOOTER STYLES
 *****************
 ****************
 */

footer {
  font-size: 1rem;
  display: flex;
  justify-content: center;
  bottom: 0;
  left: 0;
  width: 100%;
  height: 20vh;
  position: relative;
  background: #fafafa;
  z-index: 100;
}

footer .wrapper {
  display: flex;
  width: 100%;
  padding: 2rem;
  max-width: 1400px;
  align-items: center;
  justify-content: space-between;
}

@media only screen and (max-width: 649px) {
  footer .wrapper {
    flex-direction: column;
  }

  footer .wrapper h3 {
    padding-bottom: .8rem;
  }
}