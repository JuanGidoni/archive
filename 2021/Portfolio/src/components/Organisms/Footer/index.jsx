import SocialLinks from "../../Molecules/SocialLinks";
import './index.css';

const Footer = () => {
 return (
  <footer>
   <div className="wrapper">
    <h3>THANKS FOR VISITING</h3>
    <p>Â© {new Date().getFullYear()} Juan Gidoni.</p>
    <SocialLinks />
   </div>
  </footer>
 );
};
export default Footer;