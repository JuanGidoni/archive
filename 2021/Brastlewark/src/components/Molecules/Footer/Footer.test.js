import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('<Footer />', () => {
 it('renders correctly and match snapshot', () => {
  const tree = renderer
   .create(
    <Footer className="example">
     copyright example
    </Footer>
   ).toJSON();
  expect(tree).toMatchSnapshot();
 });
});