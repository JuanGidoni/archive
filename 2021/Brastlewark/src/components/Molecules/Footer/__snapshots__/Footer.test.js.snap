// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly and match snapshot 1`] = `
<footer
  className="example"
>
  copyright example
</footer>
`;
